torch/csrc/jit/frontend/concrete_module_type.h

```diff
@@ -74,14 +74,19 @@ class VISIBILITY_HIDDEN ConcreteModuleTypeBuilder {
       const TypePtr& type,
       py::object pyFunction);
 
-  void addModule(std::string name, std::shared_ptr<ConcreteModuleType> meta);
+  void addModule(
+      std::string name,
+      std::shared_ptr<ConcreteModuleType> meta,
+      TypePtr hint = nullptr);
```

Comment:
it's not clear to me from this api what `hint` is used for.

Reply:
bump

Reply:
Oops, forgot to rename, but I left a better explanation in a comment thread above. I'll add it in the comments in the code itself.